swaintek/gulpfile.js

```diff
@@ -0,0 +1,23 @@
+var gulp = require ('gulp');
+var mocha = require('gulp-mocha');
+var eslint = require('gulp-eslint')
+
+gulp.task('mocha', function () {
+  return gulp.src(['test/greet_test.js'], {read: false})
+    .pipe(mocha());
+});
+
+gulp.task('lint', function () {
+  return gulp.src(['**/*.js', '!node_modules'])
+    .pipe(eslint({
+      rules: {
+        'no-var': 1
```

Comment:
I saw you getting warning for using var - good job there.  You might want to add something to clean up the ES6 linter errors.